assets/js/view.coffee

```diff
@@ -1220,6 +1223,7 @@ window?.renderLine = renderLine
           do @render
       lineContents = renderLine lineData, lineoptions
       [].push.apply results, lineContents
+      @emit 'renderLine', row, results, options # Listeners mutate the 'results' array to change rendering #TODO: Jeff, a better interface?
```

Comment:
this is tricky.  i was thinking to have hooks that are functions taking old and returning new virtualDom elements.  eventEmitter doesn't work well for that.  also ordering of plugin hooks would matter, which is annoying

we could support additional elements much more easily, if that's the main use case (rather than arbitrary transform)